### ISSUE TITLE
why this is not compiled in new meteor 

### ISSUE THREAD

**Issue by harinanda** [gabriel-dehan/arae#1]

**Comment by gabriel-dehan** [comment]
Hey, this was more of a test application developed using meteor 4 years when we did not have a router nor a package system. Things have changed a lot since then and you would be better off not reading this code as it is severely outdated. 

If you are interested in developing something alike, please take a look at my library https://github.com/gabriel-dehan/node-tree-js/, it will allow you to handle a file system and display/move/rename/add files to a directory tree

**Comment by harinanda** [comment]
Thanks  Gabriel ,
It's great from you .

It's is great tool lack LDAP integration.

How unix user account system with LDAP açcounts into file system?.

Thanks
Nanda

**Comment by harinanda** [comment]
Hi ,
When I checked your profile .I felt great about you. I think you are
entrepreneur.
It pleasure to meet you .I once came to nice ,France like France . I am
failed entrepreneur here in India but spirit still there .

Curious know more about you

Regards
Hari nanda
.

**Comment by gabriel-dehan** [comment]
Yes my library does not handle LDAP, you could try to ensure LDAP integration through http://ldapjs.org/ though, but it is not integrated yet.
